### PR TITLE
IfcElectricAppliance Rail Import BIMQ 2021-07-21T06:47:54.2997685Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcElectricAppliance

### CODE REVIEW

File: IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT" Name="MoneyStuckRate" UniqueId="3ac99a99-4289-4850-aabe-3b9c533e43dd" PrimaryDataType="IfcRatioMeasure" />
```
Comment (jmirtsch) [inline IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/DocProperty.xml:2]: Is this a frequency measure?

File: IFC4x3/Properties/t/TicketProductionSpeed_3x1oxWIPb1Ov7kUFZmUNkg/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TicketProductionSpeed_3x1oxWIPb1Ov7kUFZmUNkg" Name="TicketProductionSpeed" UniqueId="fb072ee0-4999-4163-91ee-78f8f0797baa" PrimaryDataType="IfcReal" />
```
Comment (jmirtsch) [inline IFC4x3/Properties/t/TicketProductionSpeed_3x1oxWIPb1Ov7kUFZmUNkg/DocProperty.xml:2]: Is this a rate measure?

File: IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK" Name="TicketStuckRate" UniqueId="fcd5a049-b438-4ce1-b009-b6667ed9e694" PrimaryDataType="IfcRatioMeasure" />
```
Comment (jmirtsch) [inline IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/DocProperty.xml:2]: Is this a frequency measure?

File: IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Indicate the money stuck rate of ticket.
```
Comment (jmirtsch) [inline IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/Documentation.md:1]:
```suggestion
Indicate the frequency of money being stuck or jammed in appliance.
```

File: IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Indicate the stuck rate of tickets in the turnstile.
```
Comment (jmirtsch) [inline IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md:1]:
```suggestion
Indicate the frequency of tickets being stuck or jammed in appliance..
```

File: IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/DocProperty.xml (hunk 1 above)
Comment (czapplitec) [reply IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/DocProperty.xml:2]: It is a ratio.

File: IFC4x3/Properties/t/TicketProductionSpeed_3x1oxWIPb1Ov7kUFZmUNkg/DocProperty.xml (hunk 2 above)
Comment (czapplitec) [reply IFC4x3/Properties/t/TicketProductionSpeed_3x1oxWIPb1Ov7kUFZmUNkg/DocProperty.xml]: It is a ratio

File: IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/DocProperty.xml (hunk 3 above)
Comment (czapplitec) [reply IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/DocProperty.xml:2]: It is Ratio

File: IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/Documentation.md (hunk 4 above)
Comment (czapplitec) [reply IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/Documentation.md]: Updated, but changed "frequency" to "ratio". https://github.com/bSI-InfraRoom/IFC-Specification/pull/160/commits/8b12068ba20d88a43f933b941b15b54fce5e5c4d

File: IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md (hunk 5 above)
Comment (czapplitec) [reply IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md]: Updated, but changed "frequency" to "ratio". https://github.com/bSI-InfraRoom/IFC-Specification/pull/160/commits/8b12068ba20d88a43f933b941b15b54fce5e5c4d

File: IFC4x3/Properties/t/TicketProductionSpeed_3x1oxWIPb1Ov7kUFZmUNkg/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Indicates the producing speed of the ticket. It is measured by counting the number of tickets that can be produced per hour.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/t/TicketProductionSpeed_3x1oxWIPb1Ov7kUFZmUNkg/Documentation.md]:
```suggestion
Indicates the production speed of the ticket. It is measured by counting the number of tickets that can be produced per hour.
```

**Review by AlexBrad1eyCT — APPROVED** [review]
LFTM no issues

**Review by AlexBrad1eyCT — APPROVED** [review]

File: IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/DocProperty.xml (hunk 1 above)
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MoneyStuckRate_0woPgPGeb8KAg_EvnJFaFT/DocProperty.xml]: Propose renaiming to `MoneyStuckRatio`.

File: IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md [hunk 7]
```diff
@@ -1 +1,2 @@
-Indicates the stuck rate of tickets in the turnstile.
+Indicates the ratio of tickets being stuck or jammed in appliance or turnstile.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md]: Was this property added somewhere already? Don't see it in this PR.
```suggestion
Indicates the ratio of tickets being stuck or jammed in the appliance or turnstile.
```

File: IFC4x3/PropertyEnumerations/p/PEnum_PaymentMethod.xml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_PaymentMethod" Name="PEnum_PaymentMethod" UniqueId="03310f5f-589b-47af-92f0-53ad15bef588">
```
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/p/PEnum_PaymentMethod.xml:2]: Documentation?

File: IFC4x3/PropertyEnumerations/t/PEnum_TicketVendingMachineType.xml [hunk 9]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_TicketVendingMachineType" Name="PEnum_TicketVendingMachineType" UniqueId="49cdbd98-40ef-4c2c-befa-2f9cfbf120fe">
```
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/t/PEnum_TicketVendingMachineType.xml:2]: Documentation?

File: IFC4x3/PropertyEnumerations/v/PEnum_VendingMachineUserInterface.xml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_VendingMachineUserInterface" Name="PEnum_VendingMachineUserInterface" UniqueId="3c0d4b58-db30-475d-bbb2-57322387b989">
```
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/v/PEnum_VendingMachineUserInterface.xml:2]: Documentation?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TicketVendingMachine/Documentation.md [hunk 11]
```diff
@@ -0,0 +1 @@
+Properties of ticket vending machine. The property set can be used by _IfcElectricAppliance_ with PredefinedType _VENDINGMACHINE_.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TicketVendingMachine/Documentation.md]: Not sure how much consistency is desired, Usually italic was not used either for the entity name nor for the predefined type.
```suggestion
Properties of ticket vending machine. The property set can be used by _IfcElectricAppliance_ with PredefinedType VENDINGMACHINE.
```

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Minor changes requested.

File: IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md (hunk 7 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/t/TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK/Documentation.md]: Yes it was in IFC door for turnstiles

**Review by AlexBrad1eyCT — APPROVED** [review]

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**